app/views/organization_timeline_entries/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<% if can?(:read, @organization_timeline_entry) %>
+  <%- model_class = OrganizationTimelineEntry -%>
+  <div class="page-header">
+    <h1><%=t '.title', :default => model_class.model_name.human %></h1>
+  </div>
+
+  <dl class="dl-horizontal">
+    <dt><strong><%= model_class.human_attribute_name(:organization) %>:</strong></dt>
+    <dd><%= @organization_timeline_entry.organization.name %></dd>
+    <dt><strong><%= model_class.human_attribute_name(:entry_type).capitalize %>:</strong></dt>
+    <dd><%= @organization_timeline_entry.entry_type.capitalize %></dd>
+    <dt><strong><%= model_class.human_attribute_name(:started_at).capitalize %>:</strong></dt>
+    <dd><%= @organization_timeline_entry.started_at %></dd>
+    <dt><strong><%= model_class.human_attribute_name(:ended_at).capitalize %>:</strong></dt>
+    <dd><%= @organization_timeline_entry.ended_at %></dd>
+    <dt><strong><%= model_class.human_attribute_name(:description).capitalize %>:</strong></dt>
+    <dd><%= @organization_timeline_entry.description %></dd>
+  </dl>
+  
+  <div class="form-actions">
+    <%= link_to t('.back', :default => t("helpers.links.back")),
+              organization_timeline_entries_path, :class => 'btn btn-default' %>
+
+    <% if can?(:update, @organization_timeline_entry) %>
+      <%= link_to t('.edit', :default => t("helpers.links.edit")),
+              edit_organization_timeline_entry_path(@organization_timeline_entry), :class => 'btn btn-primary' %>
+    <% end %>
+
+    <% if can?(:destroy, @organization_timeline_entry) %>
+      <%= link_to t('.destroy', :default => t("helpers.links.destroy")),
+              organization_timeline_entry_path(@organization_timeline_entry),
+              :method => 'delete',
+              :data => { :confirm => t('.confirm', :default => t("helpers.links.confirm", :default => 'Are you sure?')) },
+              :class => 'btn btn-danger' %>
+    <% end %>
```

Comment:
There is some routing error with this link.  The link is /downtime.:id rather than downtime/:id.  The routes in this model are very weird.  It certainly wasn't designed well and could definitely use some work in the future.